src/mlpack/methods/ann/layer/noisylinear_impl.hpp

```diff
@@ -48,6 +48,48 @@ NoisyLinear<InputDataType, OutputDataType>::NoisyLinear(
   biasEpsilon.set_size(outSize, 1);
 }
 
+template<typename InputDataType, typename OutputDataType>
+NoisyLinear<InputDataType, OutputDataType>::NoisyLinear(
+  NoisyLinear&& layer) :
```

Comment:
```suggestion
    NoisyLinear&& layer) :
```

Just to match with the coding style - https://github.com/mlpack/mlpack/wiki/DesignGuidelines.